### PR TITLE
chore: visual refresh on search component

### PR DESCRIPTION
**Pull request by khamudom**
# Pull Request

## 📖 Description

using the pattern input styles

before (outline)
image
after (outline)
image

before (filled)
image
after (filled)
image

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: packages/web-components/fast-components/src/search/search.styles.ts [hunk 1]
```diff
@@ -1,226 +1,111 @@
...
+
+        ::slotted(svg) {
+            margin-inline-end: 11px;
+            margin-inline-start: 11px;
+            margin-top: auto;
+            margin-bottom: auto;
+        }
+
+        .clear-button {
+            position: absolute;
```
Comment (eljefe223) [inline packages/web-components/fast-components/src/search/search.styles.ts:82]: Please refer too fluent search box this should no longer need to be absolutely positioned. Change this to `inlne-flex` and make sure the `.input-wrapper` class is set to `flex`

**Review by EisenbergEffect — APPROVED** [review]
Approved pending Jeff's recommended change.

File: packages/web-components/fast-components/src/search/fixtures/search.html [hunk 2]
```diff
@@ -40,7 +40,13 @@ <h2>Minlength</h2>
 <!-- Start -->
 <h2>With start</h2>
 <fast-search>
-    <svg slot="start" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+    <svg
+        slot="start"
+        xmlns="http://www.w3.org/2000/svg"
+        width="16"
```
Comment (bheston) [inline packages/web-components/fast-components/src/search/fixtures/search.html:46]: I realize this is the test fixture, but for standard guidance we don't include the width and height anymore.

**Review by bheston — APPROVED** [review]
Approved with comments.

File: packages/web-components/fast-components/src/search/search.styles.ts (hunk 1 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/search/search.styles.ts]: set to inline-flex and added input-wrapper selector

**Review by eljefe223 — APPROVED** [review]

**Pull request merged**